### ISSUE TITLE
Finetuning in ocrd-train

### ISSUE THREAD

**Issue by ameera3** [OCR-D/ocrd-train#65]
In the Makefile, here is the code for finetuning

ifdef START_MODEL
$(LAST_CHECKPOINT): unicharset lists $(PROTO_MODEL)
	mkdir -p data/checkpoints
	lstmtraining \
	  --traineddata $(PROTO_MODEL) \
          --old_traineddata $(TESSDATA)/$(START_MODEL).traineddata \
	  --continue_from data/$(START_MODEL)/$(START_MODEL).lstm \
	  --net_spec "[1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c`head -n1 data/unicharset`]" \
	  --model_output data/checkpoints/$(MODEL_NAME) \
	  --learning_rate 20e-4 \
	  --train_listfile data/list.train \
	  --eval_listfile data/list.eval \
--max_iterations 10000

Why do we need the following line? I thought it was only used in training from scratch.
--net_spec "[1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx256 O1c`head -n1 data/unicharset`]" \

Should the learning rate be set lower for fine-tuning?  The learning rate for training from scratch is 20e-4, so it would seem that the learning rate for fine-tuning should be significantly lower?
--learning_rate 20e-4 \

**Comment by ameera3** [comment]
Used [https://github.com/OCR-D/ocrd-train/blob/f89efdd46c01aedea615d35e0561c50d7f86e584/Makefile](url)

**Comment by ameera3** [comment]
See [https://groups.google.com/forum/#!topic/tesseract-ocr/tmXv6U8Dviw](url)

**Comment by ameera3** [comment]
See https://groups.google.com/forum/#!topic/tesseract-ocr/tmXv6U8Dviw
